site/docs/spark-queries.md

```diff
@@ -191,57 +192,166 @@ join prod.db.table.snapshots s
+
+```sql
+SELECT * FROM prod.db.table.entries
+```
+
+<div class="markdown-table-container" markdown="block">
+| status|snapshot_id | sequence_number | data_file |
+| -- | -- | -- | -- |
+|0     |7462238160765527919|0              |{0, s3://.../dt=20211001/age=10/00000-0-38e4b886-b445-40a9-8db0-58653a331aba-00001.parquet, PARQUET, {20211001, 10}, 1, 1132, {1 -> 47, 2 -> 52, 3 -> 47, 4 -> 55}, {1 -> 1, 2 -> 1, 3 -> 1, 4 -> 1}, {1 -> 0, 2 -> 0, 3 -> 0, 4 -> 0}, {}, {1 ->    , 2 -> data1, 3 -> \n   , 4 -> 20211001}, {1 ->    , 2 -> data1, 3 -> \n   , 4 -> 20211001}, null, [4], null, 0}|
+|2     |7462238160765527919|0              |{0, s3://.../dt=20211002/age=11/00001-1-dd95f618-4738-41a4-ba0e-21bbf964d0cc-00001.parquet, PARQUET, {20211002, 11}, 1, 1132, {1 -> 47, 2 -> 52, 3 -> 47, 4 -> 55}, {1 -> 1, 2 -> 1, 3 -> 1, 4 -> 1}, {1 -> 0, 2 -> 0, 3 -> 0, 4 -> 0}, {}, {1 ->    , 2 -> data2, 3 -> \v   , 4 -> 20211002}, {1 ->    , 2 -> data2, 3 -> \v   , 4 -> 20211002}, null, [4], null, 0}|
+</div>
+
+!!! Note
+    Diffrent from metadata table [files](#files), This table exposes internal details, like files that have been deleted.
+    The field `status` in the table:
+    
+    - 0: existed 
+    - 1: added
+    - 2: deleted
+
+#### All Entries
+
+To show entries as rows from any snapshot currently tracked by the table, for both delete and data files, run:
+
+```sql
+SELECT * FROM prod.db.table.all_entries
+```
+
+<div class="markdown-table-container" markdown="block">
+| status | snapshot_id | sequence_number | data_file |
+| -- | -- | -- | -- |
+|0     |7462238160765527919|0              |{0, s3://.../dt=20211001/age=10/00000-0-38e4b886-b445-40a9-8db0-58653a331aba-00001.parquet, PARQUET, {20211001, 10}, 1, 1132, {1 -> 47, 2 -> 52, 3 -> 47, 4 -> 55}, {1 -> 1, 2 -> 1, 3 -> 1, 4 -> 1}, {1 -> 0, 2 -> 0, 3 -> 0, 4 -> 0}, {}, {1 ->    , 2 -> data1, 3 -> \n   , 4 -> 20211001}, {1 ->    , 2 -> data1, 3 -> \n   , 4 -> 20211001}, null, [4], null, 0}|
+|2     |7462238160765527919|0              |{0, s3://.../dt=20211002/age=11/00001-1-dd95f618-4738-41a4-ba0e-21bbf964d0cc-00001.parquet, PARQUET, {20211002, 11}, 1, 1132, {1 -> 47, 2 -> 52, 3 -> 47, 4 -> 55}, {1 -> 1, 2 -> 1, 3 -> 1, 4 -> 1}, {1 -> 0, 2 -> 0, 3 -> 0, 4 -> 0}, {}, {1 ->    , 2 -> data2, 3 -> \v   , 4 -> 20211002}, {1 ->    , 2 -> data2, 3 -> \v   , 4 -> 20211002}, null, [4], null, 0}|
+|0     |3830505224908377224|0              |{0, s3://.../dt=20211002/age=11/00001-1-dd95f618-4738-41a4-ba0e-21bbf964d0cc-00001.parquet, PARQUET, {20211002, 11}, 1, 1132, {1 -> 47, 2 -> 52, 3 -> 47, 4 -> 55}, {1 -> 1, 2 -> 1, 3 -> 1, 4 -> 1}, {1 -> 0, 2 -> 0, 3 -> 0, 4 -> 0}, {}, {1 ->    , 2 -> data2, 3 -> \v   , 4 -> 20211002}, {1 ->    , 2 -> data2, 3 -> \v   , 4 -> 20211002}, null, [4], null, 0}|
+</div>
+
+!!! Note
+    Like metadata table [entries](#entries), contain delete data files too, different from [all_data_files](#all-data-files)
```

Comment:
this sentence sounds a bit awkward, what about:

Like metadata table [entries](#entries), this metadata table is different from [all_data_files](#all-data-files) and contains deleted data files.